embeddings/cybertron/cybertron.go

```diff
@@ -0,0 +1,38 @@
+package cybertron
+
+import (
+	"context"
+
+	"github.com/nlpodyssey/cybertron/pkg/models/bert"
+	"github.com/nlpodyssey/cybertron/pkg/tasks/textencoding"
+	"github.com/tmc/langchaingo/embeddings"
+)
+
+// Cybertron is the embedder using Cybertron to run embedding models locally.
+type Cybertron struct {
+	encoder         textencoding.Interface
+	Model           string
+	ModelsDir       string
+	PoolingStrategy bert.PoolingStrategyType
+}
+
+var _ embeddings.EmbedderClient = (*Cybertron)(nil)
+
+func NewCybertron(opts ...Option) (*Cybertron, error) {
+	return applyOptions(opts...)
+}
+
+func (c *Cybertron) CreateEmbedding(ctx context.Context, texts []string) ([][]float32, error) {
```

Comment:
Can you document exported symbols?

Reply:
This is an implementation of the EmbedderClient interface. I can document it, but saw that in other places this wasn't done either (e.g. here: https://github.com/tmc/langchaingo/blob/main/embeddings/huggingface/huggingface.go#L32)